neurovault/apps/statmaps/tests/test_celery.py

```diff
@@ -0,0 +1,31 @@
+# in one terminal window: 
```

Comment:
I'm not sure about the image_details.html.haml, I don't think that I use it... should I delete?